src/app/mock/demo-capabilities.json

```diff
@@ -130,15 +118,61 @@
     ]
   },
   {
-    "module": "rest",
+    "module": "web-app",
+    "type": "capability",
+    "name": "Web Application",
+    "description": "Adds a simple Hello World web application to the user's project",
+    "metadata": {
+      "category": "frontend"
+    },
+    "props": [
+      {
+        "id": "framework",
+        "name": "Framework",
+        "description": "The framework to use",
+        "required": true,
+        "shared": true,
+        "type": "object",
+        "props": [
+          {
+            "id": "name",
+            "name": "Framework Name",
+            "description": "The name of the framework to use",
+            "required": true,
+            "shared": true,
+            "type": "enum",
+            "default": "react",
+            "values": [
+              "react",
+              "angular"
+            ]
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "module": "database",
     "type": "capability",
-    "name": "REST",
-    "description": "Adds REST capability to the user's project",
+    "name": "Relational Persistence",
```

Comment:
Created PR https://github.com/fabric8-launcher/launcher-creator-backend/pull/103 to apply these changes